### PR TITLE
[C++ Benchmark]: Simple fix for dynamic shapes

### PR DESCRIPTION
**Pull request by nvkevihu**
Adds `input_size` argument to replace unknown input dimensions for the synthetic data used by the C++ benchmark. Allows object detection and transformer models to be run in C++.

### CODE REVIEW

**Comment by DEKHTIARJonathan** [comment]
@meena-at-work please review

File: tftrt/benchmarking-cpp/main.cc [hunk 1]
```diff
@@ -167,6 +179,7 @@ int main(int argc, char* argv[]) {
   string model_path = "/path/to/model/";
   string signature_key = "serving_default";
   int32_t batch_size = 64;
+  int32_t input_size = 128;
```
Comment (meena-at-work) [inline tftrt/benchmarking-cpp/main.cc:182]: Hmm @nvkevihu  -- is it expected that all inputs of unknown shape will be of the same size?

Comment (nvkevihu) [reply tftrt/benchmarking-cpp/main.cc:182]: For the object detection and transformer models, yes (the obj detection models generally take square input images; e.g. 640x640, and the transformer models just need to specify sequence length). There wasn't a need to handle other shapes, so I chose to go with the simplest solution.

Comment (meena-at-work) [reply tftrt/benchmarking-cpp/main.cc:182]: sounds good.

**Comment by meena-at-work** [comment]
LGTM!

**Pull request merged**